ext/dom/node.c

```diff
@@ -1818,4 +1818,30 @@ PHP_METHOD(DOMNode, contains)
 }
 /* }}} */
 
+/* {{{ URL: https://dom.spec.whatwg.org/#dom-node-getrootnode
+Since:
+*/
+PHP_METHOD(DOMNode, getRootNode)
+{
+	zval *id;
+	xmlNodePtr thisp;
+	dom_object *intern;
+	/* Unused now because we don't support the shadow DOM nodes. Options only influence shadow DOM nodes. */
+	zval *options = NULL;
```

Comment:
Could you possibly specify this in the UPGRADING document so that one does not need to look at the source code while writing the new docs?

Reply:
Yes, will do!